### DISCUSSION TITLE
Giving a link to crates.io/crates/<CRATE> after `cargo publish`

### DISCUSSION THREAD

**Discussion by blyxyas** [rust-lang/crates.io#6111] [Feature Requests]
A lot of people want to check that their crate was published successfully after running `cargo publish`. It would be really handy if after a successful publishing process, Cargo printed the URL for checking your crate.

Maybe something like:

```
$ cargo publish
    [...]
    Updating crates.io index

Your crate was successfully published to crates.io! Check it at https://crates.io/crates/<CRATE>.

$
```
<sub>(Where `<CRATE>` gets replaced with the published crate)</sub>
This takes advantage of the URL recognition that some terminal emulators have, and it's a nice touch for first-time publishers.

**Comment by Eh2406** [comment]
This feels like it belongs in the [cargo repo](https://github.com/rust-lang/cargo). Can you open an issue there to discuss the idea?

**Reply by blyxyas** [reply]
Issue opened at rust-lang/cargo#11768; should I close this discussion then?
